docs/source/logging/logging.md

```diff
@@ -17,6 +17,21 @@ The project-side logging configuration also ensures that [logs emitted from your
 
 We now give some common examples of how you might like to change your project's logging configuration.
 
+## Using `KEDRO_LOGGING_CONFIG` environment variable
```

Comment:
You are right. I'll make it a sub-section for now. It isn't really a "project-side" configuration anymore since it's global. But at the same time, if you have multiple project you will still likely set this environment variable per project via some `.env` file or `IDE` settings.

We can re-visit this later since we haven't removed `logging` from `config_loader` yet.